CONTRIBUTING.md

```diff
@@ -1,9 +1,60 @@
-# How to contribute
+# Contributing guide
 
-Government employees, public and members of the private sector are encouraged to contribute to the repository by **forking and submitting a pull request**.
+Thanks for taking a moment and reading this guide. Is very important to have 
+everyone on the same page. This guide describes how to:
+- Get your environment ready
+- Follow our code practices
+- Run tests
+- Submitting PRs
 
 (If you are new to GitHub, you might start with a [basic tutorial](https://help.github.com/articles/set-up-git) and check out a more detailed guide to [pull requests](https://help.github.com/articles/using-pull-requests/).)
 
-Pull requests will be evaluated by the repository guardians on a schedule and if deemed beneficial will be committed to the main branch.
+All contributors retain the original copyright to their stuff, but by 
+contributing to this project, you grant a world-wide, royalty-free, 
+perpetual, irrevocable, non-exclusive, transferable license to all 
+users **under the terms of the [license](./LICENSE.md) under which 
+this project is distributed**.
 
-All contributors retain the original copyright to their stuff, but by contributing to this project, you grant a world-wide, royalty-free, perpetual, irrevocable, non-exclusive, transferable license to all users **under the terms of the [license](./LICENSE.md) under which this project is distributed**.
+## Get your environment ready
+
+### Git
+
+Make sure you have git installed on your machine. You can follow
+[this link](https://git-scm.com/downloads) for instructions.
+
+### Docker
+
+We have our environment build over Docker images. Note: things are way 
+easier if you don't need to run docker commands with sudo. Take a look
+[here](https://docs.docker.com/engine/install/) to learn how to install
```

Comment:
Sugestão: "We containerize our applications with Docker.". Me parece mais direto.

Reply:
Terminar o parágrafo com ponto final (aqui e na seção *Run tests*).

Reply:
Sugestão: colocar a nota depois da frase com o link para os guias de instalação.

Reply:
Sugestão: usar o link para a instalação do Docker Engine (https://docs.docker.com/engine/install/#server) e/ou especificar que é ele, e não o Desktop, que deve ser baixado.